client/src/css/Jessica.css

```diff
@@ -0,0 +1,55 @@
+h2 {
+  font-size: 60;
+  text-align: center;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+.name {
+  text-align: center;
+  color: #fdf1e8;
+  font-size: 70px;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  position: absolute;
+  top: 150px;
+  left: 0;
+  width: 100%;
+}
+.hobbies {
+  padding-left: 5%;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  line-height: 1.8;
+}
+.border {
+  flex: 1;
+  border-style: solid;
+  border-width: 6px;
+  border-color: #287ec7;
+  margin-right: 50px;
+  margin-left: 50px;
+  margin-bottom: 150px;
+  margin-top: 0%;
```

Comment:
pro tip: you can set the margin (and other CSS properties such as padding and border) in one line like so:
`margin: 0 50 150 50` as described [here](https://www.w3schools.com/cssref/pr_margin.asp)